app/components/view-epic-list/macro.njk

```diff
@@ -0,0 +1,24 @@
+{% macro viewEpicList(params) %}
+  {% if params.data and params.data.length and (params.type === 'met' or params.type === 'not-met') %}
```

Comment:
`params.data.length` do we need to check this is above 0? or will it infer that as falsey if it is 0

Reply:
It is inferred that it is false if 0